src/pas/plugins/oidc/controlpanel/deserializer.py

```diff
@@ -0,0 +1,79 @@
+from pas.plugins.oidc.interfaces import IOIDCControlpanel
+from plone import api
+from plone.restapi.deserializer import json_body
+from plone.restapi.deserializer.controlpanels import ControlpanelDeserializeFromJson
+from plone.restapi.deserializer.controlpanels import FakeDXContext
+from plone.restapi.interfaces import IDeserializeFromJson
+from plone.restapi.interfaces import IFieldDeserializer
+from z3c.form.interfaces import IManagerValidator
+from zExceptions import BadRequest
+from zope.component import adapter
+from zope.component import queryMultiAdapter
+from zope.interface import implementer
+from zope.interface.exceptions import Invalid
+from zope.schema import getFields
+from zope.schema.interfaces import ValidationError
+
+
+@implementer(IDeserializeFromJson)
+@adapter(IOIDCControlpanel)
+class OIDCControlpanelDeserializeFromJson(ControlpanelDeserializeFromJson):
+    @property
+    def proxy(self):
+        portal = api.portal.get()
+        plugin = portal.acl_users.oidc
```

Comment:
here it would be better to use `PLUGIN_ID` from https://github.com/collective/pas.plugins.oidc/blob/b26bba1f86e5dd2e178a74922cff34ae3b033d33/src/pas/plugins/oidc/__init__.py#L11 instead of `oidc`